src/tests/test_buildercore_utils.py

```diff
@@ -39,6 +39,10 @@ def test_isint(self):
         ]
         self.assertAllTrue(utils.isint, expected_true)
 
+    def test_isstr(self):
+        expected_true = [''] if utils.gtpy2() else ['', r'', u'']
```

Comment:
this was also passing `b''`, but that clearly fails in Python 3 not being an instance of `str`

Reply:
weird - it was all passing in python2 and 3. replicating now I can see it's not.